### PR TITLE
Added XDG appstream metadata manifest

### PR DESCRIPTION
**Pull request by xvitaly**
Added XDG appstream metadata manifest for better integration.

More info: https://freedesktop.org/software/appstream/docs/chap-Metadata.html

### CODE REVIEW

File: etc/mpv.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.mpv.Mpv</id>
```
Comment (sfan5) [inline etc/mpv.metainfo.xml:3]: where does this id come from?

File: etc/mpv.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <name>mpv</name>
+    <summary>A free, open source, and cross-platform media player</summary>
+    <description>
+        <p>Mpv is a free (as in freedom) media player for the command line. It supports a wide variety of media file formats, audio and video codecs, and subtitle types.</p>
+        <p>Mpv has an OpenGL, Vulkan, and D3D11 based video output that is capable of many features loved by videophiles, such as video scaling with popular high quality algorithms, color management, frame timing, interpolation, HDR, and more.</p>
+        <p>While mpv strives for minimalism and provides no real GUI, it has a small controller on top of the video for basic control.</p>
+        <p>Mpv can leverage most hardware decoding APIs on all platforms. Hardware decoding can be enabled at runtime on demand.</p>
```
Comment (sfan5) [inline etc/mpv.metainfo.xml:10]: The stylized name is "mpv" not "Mpv"

File: etc/mpv.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <p>Powerful scripting capabilities can make the player do almost anything. There is a large selection of user scripts on the wiki.</p>
+        <p>A straightforward C API was designed from the ground up to make mpv usable as a library and facilitate easy integration into other applications.</p>
+    </description>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-2.0+ AND LGPL-2.0-or-later</project_license>
+    <developer_name>mpv.io</developer_name>
+    <launchable type="desktop-id">mpv.desktop</launchable>
+    <translation type="gettext">mpv</translation>
```
Comment (sfan5) [inline etc/mpv.metainfo.xml:18]: this is incorrect, mpv is not translatable

File: etc/mpv.metainfo.xml (hunk 1 above)
Comment (xvitaly) [reply etc/mpv.metainfo.xml:3]: https://github.com/flathub/io.mpv.Mpv

File: etc/mpv.metainfo.xml (hunk 2 above)
Comment (xvitaly) [reply etc/mpv.metainfo.xml]: Fixed.

File: etc/mpv.metainfo.xml (hunk 3 above)
Comment (xvitaly) [reply etc/mpv.metainfo.xml]: Good catch. Fixed.

File: etc/mpv.metainfo.xml (hunk 1 above)
Comment (xvitaly) [reply etc/mpv.metainfo.xml:3]: https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Comment (sfan5) [reply etc/mpv.metainfo.xml:3]: Hmm okay. I think `io.mpv.mpv` would be better in this case.

File: meson.build [hunk 4]
```diff
@@ -1674,6 +1674,7 @@ if get_option('cplayer')
     install_data('etc/_mpv.zsh', install_dir: zsh_install_dir, rename: '_mpv')
 
     install_data('etc/mpv.desktop', install_dir: join_paths(datadir, 'applications'))
+    install_data('etc/mpv.metainfo.xml', install_dir: join_paths(datadir, 'metainfo'))
```
Comment (sfan5) [inline meson.build:1677]: shouldn't that be `io.mpv.mpv.metainfo.xml` according to the spec?

**Comment by sfan5** [comment]
Also your commit message should be more like `mpv.metainfo.xml: add XDG appstream metadata manifest`

File: meson.build (hunk 4 above)
Comment (xvitaly) [reply meson.build:1677]: It doesn't matter. But if you want to rename it, desktop file must be renamed too.

File: etc/mpv.metainfo.xml (hunk 1 above)
Comment (xvitaly) [reply etc/mpv.metainfo.xml]: Fixed.

**Comment by xvitaly** [comment]
> Also your commit message should be more like mpv.metainfo.xml: add XDG appstream metadata manifest

Done.

**Comment by sfan5** [comment]
sorry, once more: there's a trailing dot and you can remove the SoB if you want.

**Comment by xvitaly** [comment]
> sorry, once more: there's a trailing dot and you can remove the SoB if you want.

Done.

**Review by sfan5 — APPROVED** [review]

**Review by philipl — APPROVED** [review]

**Pull request merged**